core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1781,6 +1780,34 @@ export class LoggedOutUser extends BaseUser {
     await this.clickLinkAnchorToNewTab('Partner with us', partnershipsFormUrl);
   }
 
+  private async clickPartnerWithUsButtonToNewTab(
```

Comment:
We already have a function to verify destination link in the new tab. 
Maybe trying getting that Robust so to handle this case as well?

Reply:
I think, we can configure this behaviour in that function itself. But for this case wherein google translate is rejecting the request, which as per some reference might be due to  Google detecting the test's requests as unusual traffic, we can simply check and match the href of the anchor tag.